TF_Service_dotNet/TouchFree_Service/Program.cs

```diff
@@ -21,6 +21,12 @@ static void Main(string[] args)
             Host.CreateDefaultBuilder(args)
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
+                    //webBuilder.ConfigureAppConfiguration((hostingContext, config) =>
+                    //{
+                    //    config.AddJsonFile("interaction-tuning.json",
+                    //                       optional: true,
+                    //                       reloadOnChange: true);
+                    //});
```

Comment:
Is this still needed or can it be deleted?

Reply:
I have removed this for now